modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -337,4 +470,39 @@ protected function assertElementsTexts(array $expected, array $elements): void {
     ));
   }
 
+  /**
+   * Create a list page node with filters configured.
+   *
+   * @param string $bundle
+   *   Nodes of this bundle will be listed.
+   * @param array $exposed_filters
+   *   Facet machine names linked to the bundle's facet source.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   The list page node created.
+   */
+  protected function createListPage(string $bundle, array $exposed_filters): NodeInterface {
+    $list_page = Node::create([
+      'type' => 'oe_list_page',
+      'title' => 'Results',
```

Comment:
I think 'Results' is confusing as a node title here.
```suggestion
      'title' => 'Example list page',
```